src/spaniel-observer.ts

```diff
@@ -129,8 +138,11 @@ export class SpanielObserver {
     }
   }
   private flushQueuedEntries() {
-    this.callback(this.queuedEntries.slice());
-    this.queuedEntries = [];
+    let queued = this.queuedEntries.slice();
```

Comment:
I feel like at one point I had a reason but can no longer remember, will try just returning the queue and reseting. The queue items are static, there's no function to be executed.